tree/tree/src/TTreeCache.cxx

```diff
@@ -613,11 +638,253 @@ Int_t TTreeCache::DropBranch(const char *bname, Bool_t subbranches /*= kFALSE*/)
    return res;
 }
 
+///////////////////////////////////////////////////////////////////////////////
+//// Start of methods for the miss cache.
+///////////////////////////////////////////////////////////////////////////////
+
+/**
```

Comment:
The doc start pattern for functions should be
```
////////////////////////////////////////////////////////////////////////////////
///
``` rather than C style and the content should be useable by doxygen.